scripts/ci/get_operator_github_branch.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash -xe
+set +o pipefail
+
+does_operator_branch_has_image=false
+operator_branch=develop
+DOCKER_HUB_USERNAME=csiblock1
+DOCKER_HUB_PASSWORD=$csiblock_dockerhub_password
+wanted_image_tag=`echo $CI_ACTION_REF_NAME | sed 's|/|.|g'`
```

Comment:
this logic is starting to have [too](https://github.com/IBM/ibm-block-csi-driver/blob/364b42bf5f7215c0e1bc9700ead61d9917c44351/.github/workflows/driver_ci.yaml#L106) [many](https://github.com/IBM/ibm-block-csi-driver/blob/364b42bf5f7215c0e1bc9700ead61d9917c44351/scripts/ci/get_operator_github_branch.sh#L29) [duplications](https://github.com/IBM/ibm-block-csi-driver/blob/a538b5ae9ee1822ed702671aa09fdbd9965ce7c3/scripts/ci/build_push_images.sh#L10).
please try to move it to a single common location

Reply:
yes please